.github/workflows/release.yml

```
@@ -177,6 +255,9 @@ jobs:
             
             ## Installation
             
+            ### Desktop Extension (Recommended)
+            Download the `.dxt` file below and import it into your AI desktop application.
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)